utilities/rpp-unittests/HOST_NEW/Tensor_host_pkd3.cpp

```diff
@@ -103,83 +105,160 @@ int main(int argc, char **argv)
 
     if (verbosity == 1)
     {
-        printf("\nInputs for this test case are:");
-        printf("\nsrc1 = %s", argv[1]);
-        printf("\nsrc2 = %s", argv[2]);
-        printf("\ndst = %s", argv[3]);
-        printf("\nu8 / f16 / f32 / u8->f16 / u8->f32 / i8 / u8->i8 (0/1/2/3/4/5/6) = %s", argv[4]);
-        printf("\noutputFormatToggle (pkd->pkd = 0 / pkd->pln = 1) = %s", argv[5]);
-        printf("\ncase number (0:86) = %s", argv[6]);
+        if (test_type == 0)
+        {
+            printf("\nInputs for this test case are:");
+            printf("\nsrc1 = %s", argv[1]);
+            printf("\nsrc2 = %s", argv[2]);
+            printf("\ndst = %s", argv[3]);
+            printf("\nu8 / f16 / f32 / u8->f16 / u8->f32 / i8 / u8->i8 (0/1/2/3/4/5/6) = %s", argv[4]);
+            printf("\noutputFormatToggle (pkd->pkd = 0 / pkd->pln = 1) = %s", argv[5]);
+            printf("\ncase number (0:86) = %s", argv[6]);
+            printf("\nUNIT/PERFORMANCE - 0/1 = %s", argv[8]);
+        }
+        else
```

Comment:
There is lot of common part in this if else case. Move the common part outside this if else add if else only for the uncommon part